Makefile

```diff
@@ -487,3 +487,16 @@ dev-release:
 validate-release: setup-venv
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install yq==2.10.0
 	BUNDLE_VERSION=$(BASE_BUNDLE_VERSION) CHANNEL="beta" ./hack/validate-release.sh
+
+# Operator-sdk release version
+RELEASE_VERSION ?= v0.16.0
+
+.PHONY: install-operator-sdk
+install-operator-sdk:
+ifeq ($(shell uname -s), Darwin)
+	curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin
+	chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin /usr/local/bin/operator-sdk && rm operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin
```

Comment:
current logic install it under `$GOPATH/bin`, let's keep it like this, instead of installing it globally for the system.

Reply:
`GOPATH/bin` is not necessarily on my PATH when I am working with the project. Ideally, I would not have to install a specific version and the copy of the SDK I receive from brew would work when I am working locally. But the project has neglected to keep up with the SDK releases and here we are.

Reply:
Ok, in that case, how about that we install operator-sdk in project `out` dir and use that copy throught the project? Or to have Dockerfile crafted with the proper operator-sdk version and spin container whenever we need operator-sdk?

Reply:
The assumption was that `operator-sdk` was provided by the environment, not managed by project's build system, which fits @gorkem's requirement on using an `operator-sdk` provided by `brew`.

Unfortunately the presented approach overwrites the host's toolchain, which I consider harmful.

A different approach would be to assert whether the system's `operator-sdk`'s version matches the required version for building the operator in the Makefile and offer the `hack/install-operator-sdk.sh` script to make it easier to install it in the developer's host, leaving the responsibility to choose a specific version to the host administrator, like it currently is.